.github/workflows/pull_request.yml

```diff
@@ -10,4 +10,5 @@ jobs:
     with:
       branch: ${{github.head_ref}}
       gradlew-command: assembleDevelop
+      build-debug-tests: true
```

Comment:
build test app has been added to the PR workflow to identify changes in functions signature early